hack/apidiff.sh

```diff
@@ -149,6 +149,10 @@ run () {
     out="$1"
     mkdir -p "$out"
     for d in "${targets[@]}"; do
+        if ! [ -d "${d}" ]; then
+            echo "module ${d} does not exist, skipping ..."
+            continue
+        fi
```

Comment:
a module can be added or removed, if that happens in the PR the script will fail and exit early because it will not be able to find it